docs/content/preview/benchmark/resilience/jepsen-testing.md

```diff
@@ -0,0 +1,210 @@
+---
+title: Jepsen testing
+headerTitle: Jepsen testing
+linkTitle: Jepsen testing
+description: Description of Jepsen testing.
+menu:
+  preview:
+    identifier: jepsen-testing-1
+    parent: resilience
+    weight: 13
+aliases:
+  - /preview/benchmark/resilience/jepsen-testing-ysql/
+  - /preview/benchmark/resilience/jepsen-testing-ycql/
+type: docs
+---
+
+
+This documentation page contains descriptions of Jepsen tests that YugabyteDB runs daily for each
+major release currently available, including master branch releases.
+
+## Scenarios
+
+### Bank
+
+This test simulates money transfer between accounts; it uses a table of form T`(id int PRIMARY KEY,
+balance bigint)`. The workload performs transactional transfers between accounts - `(UPDATE T
+balance = balance - x WHERE id = ?; UPDATE T SET balance = balance + x WHERE id = ?;)` as well as
+reads of the whole table. It expects the grand total to remain the same, both in short and long
+term.
+
+![Load Phase Results](/images/benchmark/jepsen/jepsen-1-bank.png)
+
+```plpgsql
+BEGIN TXN;
+UPDATE T SET balance = balance + x WHERE id = ?;
+UPDATE T SET balance = balance - x WHERE id = ?;
+COMMIT;
+```
+
+### Bank-contention (YSQL)
+
+In addition to UPDATE transactions, there are also INSERT or INSERT/DELETE transactions, all
```

Comment:
```suggestion
In addition to UPDATE transactions, there are also INSERT or INSERT/DELETE transactions, all operating under the assumption that the overall `SUM(balance)` must remain consistent. For simplicity in the case of Jepsen, 5 keys are designated for UPDATE operations, and 5 for random INSERTs and DELETEs.
```

Reply:
Is it our guideline to put paragraph into a single line?
It is very confusing to fix something in this case in source code, since it is not on the screen. As far as I understand it does not affect target formatting

Reply:
Fixed all occurences